hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/MirrorProperties.java

```diff
@@ -53,9 +55,10 @@ public class MirrorProperties {
     @NotNull
     private Map<String, MigrationProperties> migration = new CaseInsensitiveMap<>();
 
-    @NotNull
-    private HederaNetwork network = HederaNetwork.DEMO;
+    @NotBlank
+    private String network = HederaNetwork.DEMO;
 
+    @Setter(AccessLevel.NONE) // Not user set-able
     private String networkPrefix;
```

Comment:
This is currently only used by tests. When we add resettable support, even then it would be called rarely when we detect a potential stream reset and may not be worth caching here and might be fine re-calculating in a `getNetworkPrefix()`. If we really think it's worth caching, can simplify:
```suggestion
    @Getter(lazy = true)
    private String networkPrefix = StringUtils.substringBefore(network, '-');
```

Reply:
Taking a closer look at lombok lazy getter, it fires off only the first time it's called and then caches that result. Thus if `setNetwork()` is called again this getter will not reflect any changed network prefix. But in the normal course of reading properties from `application.yml` or elsewhere, this won't be an issue.

You're right though, #5713 is where any network prefix would be utilized. This PR just provided some basic String manipulation to accommodate and makes available any prefix specification; presently only utilized in the tests.

Reply:
It's probably easiest to just compute on read via a `getNetworkPrefix()` and no field.

Reply:
I concur. It's shaping up to be something like the following, using a single `network` field with a lombok generated setter.
```
    public String getNetwork() {
        return StringUtils.substringBefore(this.network, NETWORK_PREFIX_DELIMITER).toLowerCase();
    }

    public String getNetworkPrefix() {
        var networkPrefix = StringUtils.substringAfter(this.network, NETWORK_PREFIX_DELIMITER);
        return StringUtils.isEmpty(networkPrefix) ? null : networkPrefix;
    }
```
`StringUtils.substringAfter()` returns an empty string if the delimiter is not found. I prefer a `null` to that, or perhaps `getNetworkPrefix()` could return `Optional<String>`?

Reply:
Done